.github/workflows/release.yml

```diff
@@ -55,10 +55,20 @@ jobs:
         uses: cachix/cachix-action@v14
         with:
           name: k-framework
+          extraPullNames: k-framework-binary
           authToken: '${{ secrets.CACHIX_PUBLIC_TOKEN }}'
 
-      - name: Build
-        run: nix build .#kore-exec .#kore-rpc-booster
+      - name: 'Build and Cache'
+        uses: workflow/nix-shell-action@v3
```

Comment:
This seems to introduce an error message `unrecognised flag '-c'` (cannot possibly come from the commands below, and happens before any build output is printed). Is this action stable/recommended by people?

Reply:
Which looks like also produces the same error. Not sure when this was introduced though without looking at git history seems it didn't raise any flags though. I'll spend a minute on it see if there' and a similar error is thrown on linux as well.